standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/common/metrics/metrics2/CodahaleMetrics.java

```diff
@@ -414,6 +415,9 @@ private boolean initCodahaleMetricsReporterClasses() {
       try {
         // Note: Hadoop metric reporter does not support tags. We create a single reporter for all metrics.
         Constructor<?> constructor = name.getConstructor(MetricRegistry.class, Configuration.class);
+        if (name.getName().equals(Metrics2Reporter.class.getName())) {
+          DefaultMetricsSystem.shutdown();
```

Comment:
please put a code comment about this to clarify: why it's needed

Reply:
Sure